staging/src/slime.io/slime/modules/limiter/controllers/envoy_filters.go

```diff
@@ -70,9 +71,10 @@ func (r *SmartLimiterReconciler) GenerateEnvoyConfigs(spec microservicev1alpha2.
 	}
 
 	sets = append(sets, &networking.Subset{Name: util.WellknownBaseSet})
-	svcSelector := generateServiceSelector(r, params)
-	if len(svcSelector) == 0 {
-		log.Info("get empty svc selector base on %v", params)
+
+	svcSelector, err := generateServiceSelector(r, params)
+	if err != nil {
+		log.Info("get svc selector err base on %v", params)
 		return setsEnvoyFilter, setsSmartLimitDescriptor, globalDescriptors, nil
```

Comment:
how to handle err?

Reply:
if err , record log  and  will not genenrate envoyfilter

and now it will record log and return err instead of swallowing it